src/components/avatar-creator-raw.tsx

```diff
@@ -43,17 +44,17 @@ export const AvatarCreatorRaw: FC<AvatarCreatorRawProps & EventReceivedProps> =
     );
   };
 
-  const subscribe = (event: any) => () => {
-    const iframeEvent = JSONTryParse<IFrameEvent<AvatarCreatorEvent>>(event);
+  const subscribe = (event: MessageEvent) => {
+    const avatarCreatorEvent = JSONTryParse<AvatarCreatorEvent>(event.data);
 
-    if (iframeEvent?.source !== RPM_TARGET) return;
+    if (avatarCreatorEvent?.source !== RPM_TARGET) return;
 
-    if (iframeEvent?.eventName === IFRAME_READY_EVENT) {
+    if (avatarCreatorEvent?.eventName == IFRAME_READY_EVENT) {
```

Comment:
why not strict equal? 🤔

Reply:
Ah, good spot! I've accidentally left this in after trying to investigate why events were not being received. Will remove 👍